core/src/test/java/org/apache/iceberg/view/ViewCatalogTests.java

```diff
@@ -127,6 +127,36 @@ public void basicCreateView() {
     assertThat(catalog().viewExists(identifier)).as("View should not exist").isFalse();
   }
 
+  @Test
+  public void defaultViewProperties() {
+    TableIdentifier identifier = TableIdentifier.of("ns", "view");
+
+    if (requiresNamespaceCreate()) {
+      catalog().createNamespace(identifier.namespace());
+    }
+
+    assertThat(catalog().viewExists(identifier)).as("View should not exist").isFalse();
+
+    String location =
```

Comment:
this shouldn't be required to test default view props, so let's omit setting the location